### PR TITLE
fix(node): switch to node 6 for container and testing

### LINKED ISSUE #95: Upgrade to node 6?
I was going to rev this, but it's still on node 4. Node 4 support goes away in April 2018. Should we look at upgrading this to node 6, or is it possible that we won't be using this by then?

cc @rfk

### PR DESCRIPTION
**Pull request by vladikoff**
Fixes #95 

@jbuck r?

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM node:4.8.5-slim
```
Comment (jbuck) [inline Dockerfile]: This should be 6.11.5-slim

File: package.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
   "license": "MPL-2.0",
   "name": "browserid-verifier",
   "description": "A node.js verification server for BrowserID assertions.",
-  "version": "0.8.2",
```
Comment (jbuck) [inline package.json]: This shouldn't be revved until this patch lands

File: Dockerfile (hunk 1 above)
Comment (vladikoff) [reply Dockerfile]: 👍

File: package.json (hunk 2 above)
Comment (vladikoff) [reply package.json]: 👍

**Pull request merged**